.github/workflows/pipeline.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       fail-fast: false
       matrix:
         # The .x indicates "the most recent one"
-        node: [19.x, 18.x, 17.x, 16.x, 14.x, 14.17.0, 12.x, 12.22.0]
```

Comment:
Dropped support for Node.js versions lower than 16. They're not maintained long time ago and doesn't support package.json `exports` which is used at least in `@typescript-eslint`.